jest.config.js

```diff
@@ -11,7 +11,7 @@ module.exports = {
   },
   transformIgnorePatterns: [
     '<rootDir>/node_modules/(?!(ol|@babel|jest-runtime|@terrestris|color-*|query-string|' +
-    'decode-uri-component|split-on-first|filter-obj))'
+    'decode-uri-component|split-on-first|filter-obj|geostyler-openlayers-parser|geostyler-style))'
```

Comment:
Is the geostyler even a dependency of this package?

Reply:
At least a transitive one, the `@terrestris/ol-util` has a direct dependency to it.